src/cartservice/.dockerignore

```diff
@@ -0,0 +1,8 @@
+**/*.sh
+**/*.bat
+tests/
+**/bin/
+**/obj/
+**/out/
+Dockerfile*
+docker-compose*
```

Comment:
similarly, here?

Reply:
Those files should not go the container itself, so to avoid any `COPY` command in the `Dockerfile` bringing them, we could add their exclusion in the `.dockerignore` file as a best practice.